### PR TITLE
pip

### PR DESCRIPTION
**Pull request by jjhelmus**
Rebase of @msarahan PR, #552, to pick up new CI settings.
Updated recipe to pip version 8.1.2

### CODE REVIEW

**Comment by jjhelmus** [comment]
Hold off on merging this until the conda-smithy produces feedstocks with `add_pip_as_python_dependency: false` in the .condarc file.

**Comment by jakirkham** [comment]
So, `conda-smithy`'s behavior has been changed so that feedstocks are created with `add_pip_as_python_dependency: false`. Is there any other reason this is WIP or can that be removed?

**Comment by jakirkham** [comment]
Is there something more that needs to be done here, @jjhelmus, or can it be merged?

**Comment by jjhelmus** [comment]
Yes this should be good to go!

**Comment by jakirkham** [comment]
Thanks for clarify. Removed the "WIP" and will merge as soon as we clear some stuff up from the staging area.

File: recipes/pip/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  home: https://pip.pypa.io/en/stable/
+  license: MIT
+  license_file: LICENSE.txt
+  summary: PyPA recommended tool for installing Python packages
+  doc_url: https://pip.pypa.io/en/stable/
+  dev_url: https://github.com/pypa/pip
+
+extra:
+  recipe-maintainers:
+    - msarahan
```
Comment (jakirkham) [inline recipes/pip/meta.yaml]: Maybe we should add ourselves here too, @jjhelmus? I'm guessing the usual gang is interested @pelson, @ocefpaf, yourself, and me. Though I'm ok with doing this in the feedstock too. Either way.

**Comment by jakirkham** [comment]
Maybe more maintainers, but that isn't a blocker. LGTM. Any final feedback @conda-forge/core?

File: recipes/pip/build.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```
Comment (pelson) [inline recipes/pip/build.sh]: Should this be single-version-externally managed?

Comment (jakirkham) [inline recipes/pip/build.sh]: Yeah, it uses `setuptools`.

So let's add the arguments `--single-version-externally-managed --record record.txt`.

File: recipes/pip/bld.bat [hunk 3]
```diff
@@ -0,0 +1,9 @@
+python setup.py install
```
Comment (jakirkham) [inline recipes/pip/bld.bat]: Also add the arguments `--single-version-externally-managed --record record.txt` here too.

**Comment by pelson** [comment]
Happy to be added as a maintainer.

**Pull request merged**